modules/javafx.graphics/src/main/java/com/sun/javafx/scene/PointLightHelper.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2016, 2019, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Update the "last modified year" for 2020 or else revert (this applies to all copyright year updates).

Reply:
Can the script not get it?

Reply:
Yes, it can. So it would be fine to leave it.